### PR TITLE
Feat/docs  content section

### LINKED ISSUE #1570: [Website] Content section
## v2

This page is using the Component template, but will likely need to be fully re-written due to the breaking changes in v2

### Intro
Likely want to rephrase the intro paragraph, we don't want to mention content block (internal only). Could mention it's the main building block(?) or base level component that accept a number of children to build flexible "sections" or groupings throughout the page. 

>Content section is a high-level content component that acts as the main content unit to build web page experiences for IBM.com. It encapsulates other content components (one or multiple Content blocks and/or Content groups) and brings them together, providing flexibility to create many different solutions for users.

### Resources
- [ ] Update links

### Overview
Re-do
- New paragraph
- New imagery
- Remove all existing images
- New when to use / not to use

### Behaviors
- New images needed

**Examples**
- This GIF is a good example of how we could showcase how the content section can accept multiple children. Wish it wasn't on the g10 background. 

### Tips & techniques
- Re-do
   - first p is referencing responsive behavior
   - second p will no longer be applicable 
   - third p – I don't like referencing the components as "units"
   - fourth p review if we will have CTA stack of if adopters should add their own CTAs

### Content guidance
- [ ] Check Character counts still stand
- [ ] Update with additional fields

### Related components
- [ ] Remove "Related Components' section

---

### Acceptance criteria [TBD]
- [ ] Check if a [page design exists]((https://ibm.ent.box.com/folder/151141324220)) to leverage new images 
- [ ] Utilize the [Component page template](https://ibm.ent.box.com/folder/147223579360)
   - [ ] If a page is missing chunks of the template, make a new issue to address the missing pieces
- [ ] Utilize the Image production kit to create new images
- [ ] Resources links are cleaned up
- [ ] Page must be reviewed by a team member

### PR DESCRIPTION
**Pull request by oliviaflory**
### Related Ticket(s)

Closes #1570 

### Description

Major update to Content section page

### Changelog

**New**

- Complete overhaul of content
   - New images
   - New content

### CODE REVIEW

File: src/pages/components/content-section.mdx [hunk 1]
```diff
@@ -14,146 +11,175 @@ import ResourceLinks from 'components/ResourceLinks';
...
+
+The content section has two types of optional headings – **content section** and **content block** – that can be turned on and off to fit the layout needs.
+
+![Image of the content section](../../images/component/content-section/content-section-modifier-heading-01.png)
+
+<Caption>Example of the content section.</Caption>
+
+#### Content section heading
+
+Turn off the content section heading to utilize the vertical table of contents component for long form reading experiences that allow the user to quickly navigate through the content.
```
Comment (RichKummer) [inline src/pages/components/content-section.mdx:56]:
```suggestion
Turn off the content section heading to utilize the vertical [table of contents](../components/table-of-contents) component for long form reading experiences that allow the user to quickly navigate through the content.
```

Adding a link here

File: src/pages/components/content-section.mdx [hunk 2]
```diff
@@ -14,146 +11,175 @@ import ResourceLinks from 'components/ResourceLinks';
...
+<Column colMd={4} colLg={6}>
 
-![A visual example of the hierarchy of different content components](../../images/component/content-section/component--content-section--structure.gif)
+![Example highlighting content section heading](../../images/component/content-section/content-section-modifier-heading-02.png)
 
 <Caption>
-  See how Content section, Content block, Content group and Content item act as
-  hierarchical content containers for each other.
+  Example of the content section component, where the content section heading is
+  highlightedp.
```
Comment (RichKummer) [inline src/pages/components/content-section.mdx:66]:
```suggestion
  highlighted.
```

File: src/pages/components/content-section.mdx [hunk 3]
```diff
@@ -14,146 +11,175 @@ import ResourceLinks from 'components/ResourceLinks';
...
-Do not use a Content section to separate only content items, or other related basic content items (like paragraphs)
-inside a web page. Use a [Content group](https://www.ibm.com/standards/carbon/components/content-group) instead.
+<Column colMd={4} colLg={6}>
 
-## Behaviors
+![Example highlighting content block heading](../../images/component/content-section/content-section-modifier-heading-04.png)
+
+<Caption>
+  Example of the content section component, where the content block heading is
+  highlightedp.
```
Comment (RichKummer) [inline src/pages/components/content-section.mdx:96]:
```suggestion
  highlighted.
```

File: src/pages/components/content-section.mdx [hunk 4]
```diff
@@ -14,146 +11,175 @@ import ResourceLinks from 'components/ResourceLinks';
...
-[Content block simple](https://www.ibm.com/standards/carbon/components/content-block-simple), or a
-[Content block cards](https://www.ibm.com/standards/carbon/components/content-block-cards), both ready-to-use
-components.
+![Image of a content section with an 12 column child component](../../images/component/content-section/content-section-modifier-children-01.png)
+
+<Caption>
+  Example of the content section with a child component that spans 12 columns.
+</Caption>
+
+There is no limit to how many child components the content section can accept, consider the story you are trying to tell, the overall hierarchy of the page and how many content sections should be used to break up the content. View the [gallery](#gallery) to see examples of content section with various child components.
```
Comment (RichKummer) [inline src/pages/components/content-section.mdx:138]:
```suggestion
There is no limit to how many child components the content section can accept - consider the story you are trying to tell, the overall hierarchy of the page and how many content sections should be used to break up the content. View the [gallery](#gallery) to see examples of content section with various child components.
```

File: src/pages/components/content-section.mdx [hunk 5]
```diff
@@ -14,146 +11,144 @@ import ResourceLinks from 'components/ResourceLinks';
...
+  Example of the content section with a child component that spans 12 columns.
+</Caption>
 
-#### Desktop breakpoints
+#### Mixing children widths
 
-If the browser is wider than the max breakpoint of 1584px, the Content section and all other page content will center
-and extra margin will appear on either side of the page layout. The Content section heading, description and link CTA
-will take the first four (out of 16) columns of the grid, while the children container will take the other 12 columns.
+The content section can support mixing of various children and how many columns of the grid the children can span. Most of the Carbon for IBM.com components have been carefully designed with line length, and column usage in mind, and therefore should fit within the columns without alteration.
```
Comment (RichKummer) [inline src/pages/components/content-section.mdx:71]:
```suggestion
The content section can support mixing of various children and how many columns of the grid the children can span. Most of the Carbon for IBM.com components have been carefully designed with line length and column usage in mind, and therefore should fit within the columns without alteration.
```

Comment (RichKummer) [reply src/pages/components/content-section.mdx:71]: New child content looks good! Just made a tiny edit for clarity

**Review by IgnacioBecerra — APPROVED** [review]
LGTM! Just gotta deal with the merge conflict.

**Pull request merged**